tests/python/unittest/test_loss.py

```diff
@@ -185,27 +185,28 @@ def test_l1_loss():
 @with_seed(1234)
 def test_ctc_loss():
     loss = gluon.loss.CTCLoss()
-    l = loss(mx.nd.ones((2,20,4)), mx.nd.array([[1,0,-1,-1],[2,1,1,-1]]))
+    l = loss(mx.nd.ones((2, 20, 4)), mx.nd.array([[1, 0, -1, -1], [2, 1, 1, -1]]))
```

Comment:
As discussed, please address pylint issue in another PR, it should be going to master first before this PR